docs/main/prerequesites.md

```diff
@@ -13,7 +18,7 @@ The following operating systems are **officially** supported:
 
 | Distribution | Release          | Architecture        |
 | ------------ | ---------------- | ------------------- |
-| Raspbian     | Jessie / Stretch | ARM                 |
+| Raspbian     | Jessie / Stretch/Buster | ARM                 |
```

Comment:
Might want to keep consistent formatting by adding some spaces here.

Reply:
Thanks, will fix.